### ISSUE TITLE
[Testing] - Separate axe testing so best practices warns instead of fails

### ISSUE THREAD

**Issue by mejiaj** [uswds/uswds#3333]
## Description

Right now best practices causes tests to fail. For example, [this](https://circleci.com/gh/uswds/uswds/4595?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link) build complains about multiple banner landmark when it's [valid according to w3c](https://www.w3.org/TR/2017/NOTE-wai-aria-practices-1.1-20171214/examples/landmarks/banner.html).

We should first test for `'section508', 'wcag2a', 'wcag2aa'` in `axe-tester.js` and fail on errors. But warn on `"best-practice"` instead of failing so users are aware of the difference.

## Steps to reproduce the issue

1. Go to `axe-tester.js:20` and make sure the you have the following:
`values: ['section508', 'wcag2a', 'wcag2aa', 'best-practice']`
2. Run `npm run test:ci`
3. See it fail because of multiple banner landmarks
4. Remove `best-practice` and test again

**Comment by mejiaj** [comment]
There seems to be an open issue related to this:
https://github.com/dequelabs/axe-core/issues/2186

**Comment by mejiaj** [comment]
@sarnikbixal we have a test that checks for accessibility issues (you can manually run that with `npm run test` or `npm run test:ci`).

If you want to just run the accessibility tests do `gulp regression`. It runs a headless chrome instance and the script `axe-tester.js`.

## What we want to do
We want to run 2 checks for accessibility, 1 should check for `"section508", "wcag2a", "wcag2aa"` and error if they fail against those. The other should check later for `"best-practice"` and give a `warning`.

The reason behind this is if it errors and fails on `best-practice` it blocks your PR from merging.

**Comment by sarnikbixal** [comment]
Documentation is lacking. This is a pre-build rule set. We might need to modify it within - but again no documentation. We will have to trial and error test this if we proceed.
